### PR TITLE
Pull the tile content into directives so unserviced things can use them

### PR DESCRIPTION
**Pull request by jwforres**
Among other things:
Fixes https://github.com/openshift/origin-web-console/issues/83

unserviced_stuff

### CODE REVIEW

**Comment by jwforres** [comment]
@spadgett fixes everything except notifications on unserviced stuff, would rather do that as a follow-up so these changes can get in to minimize merge conflicts

I am hitting a weird issue, i think affected by loading order because it only shows up some of the time.  Its also only on an unserviced DC:
missing_donut

**Comment by jwforres** [comment]
Ok fixed that issue.  PTAL @spadgett

**Comment by jwforres** [comment]
hmm travis complaining about the dist, will have to check into that, but you can go ahead and review the rest

File: app/views/overview/_pod.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  <div class="rc-header"> <!-- TODO may want different treatment for a pod-name? -->
+    Pod:
+    <a ng-href="{{pod | navigateResourceURL}}">{{resource.metadata.name}}</a>
+  </div>
+  <div row class="deployment-body">
+    <div column class="overview-donut">
+      <pod-donut pods="[pod]"></pod-donut>
+    </div>
+    <div column class="deployment-details">
+      <metrics ng-if="showMetrics" pod="pod" profile="compact" class="overview-metrics"></metrics>
```
Comment (spadgett) [inline app/views/overview/_pod.html]: `ng-if="showMetrics && !collapse`?

File: app/views/overview/_rc.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+          hpa="getHPA(deployment.metadata.name, dcName)"
+          limit-ranges="limitRanges"
+          scalable="isScalableDeployment(deployment)"
+          alerts="alerts">
+      </deployment-donut>
+    </div>
+    <!-- /all visible deployments -->
+    <!-- metrics or pod template -->
+    <div column class="deployment-details">
+      <metrics ng-if="showMetrics" deployment="deployment" profile="compact" class="overview-metrics"></metrics>
```
Comment (spadgett) [inline app/views/overview/_rc.html]: `ng-if="showMetrics && !collapse`?

File: app/scripts/directives/overview/dc.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+        $scope.viewPodsForDeployment = function(deployment) {
+          if (_.isEmpty($scope.podsByDeployment[deployment.metadata.name])) {
+            return;
+          }
+
+          Navigate.toPodsForDeployment(deployment);
+        };
+
+        $scope.getHPA = function(rcName, dcName) {
```
Comment (spadgett) [inline app/scripts/directives/overview/dc.js]: Wonder if we should move this to overview.js so we don't have to repeat it here and in overview/service.js.

File: app/views/overview.html [hunk 4]
```diff
@@ -43,10 +43,29 @@
               </div>
 
               <!-- Unserviced DCs, RCs, and pods -->
-              <div>
-                <overview-service-group
-                  ng-if="(monopodsByService[''] | hashSize) > 0 || (deploymentConfigsByService[''] | hashSize) > 0 || (deploymentsByService[''] | hashSize) > 0">
-                </overview-service-group>
+              <div row wrap ng-if="(monopodsByService[''] | hashSize) > 0 || (deploymentConfigsByService[''] | hashSize) > 0 || (deploymentsByService[''] | hashSize) > 0">
+                <!-- unserviced DC -->
+                <div ng-repeat="(dcName, deploymentConfig) in deploymentConfigsByService['']"
```
Comment (spadgett) [inline app/views/overview.html]: `track by (deploymentConfig | uid)`?

**Comment by jwforres** [comment]
@spadgett updated, and fixed the issue with the dist i believe, my deps were out of date

**Comment by jwforres** [comment]
[merge]

**Pull request merged**